src/components/UI/Tab.vue

```diff
@@ -89,31 +92,48 @@ export default defineComponent({
       window.removeEventListener('resize', handleResize)
     })
 
-    return { setTabRef, lineStyle, changeTab }
+    return { setTabRef, lineStyle, changeTab, selectedTab }
   }
 })
 </script>
 
 <style lang="scss" module>
 .tabWrapper {
-  position: relative;
+  position: sticky;
+  top: -1.5em;
+  color: $ui-secondary;
+  background: $bg-primary;
+  box-shadow: 0px 10px 10px $bg-primary;
   display: flex;
-  justify-content: center;
-  border-bottom: solid 1px;
+  z-index: 10;
+  justify-content: left;
+  border-bottom: solid 1px $ui-secondary;
+  @include size-body;
+  @include weight-bold;
   .tab {
-    font-size: 1.2rem;
-    padding: 0.5rem 1rem;
+    padding: 0.75rem 3rem;
+    margin: 0.125rem 0;
+    border-radius: 100rem;
     cursor: pointer;
     &:hover {
-      color: #cfb998;
+      color: $ui-white;
+      background: $bg-primary-highlight;
       transition: color 0.2s;
```

Comment:
```suggestion
      transition: all 0.2s;
```
で`.tab`につけたほうがよさそう(background-colorにきかない、selectedTabの変更に聞かないため)